packages/server/src/listener.js

```diff
@@ -75,8 +75,16 @@ export default class Listener {
     // Call server.listen
     try {
       this.server = await new Promise((resolve, reject) => {
-        this._server.on('error', error => reject(error))
-        const s = this._server.listen(listenArgs, error => error ? reject(error) : resolve(s))
+        this._server.on('error', reject)
+        const s = this._server.listen(listenArgs, (error) => {
+          s.removeListener(reject)
```

Comment:
I don't think this will work as written.

How about this?
```suggestion
          this._server.off('error', reject)
```

Reply:
Alternatively, we can use server.once i guess

Reply:
Yes, that's nicer.